api/http/internal.go

```diff
@@ -33,8 +34,7 @@ const (
 type InternalHandler APIHandler
 
 // POST /tenants/:tenant_id/devices
-// code: 202 - device provisioned to iothub
-//       204 - nothing happened
+// code: 204 - device provisioned to iothub
```

Comment:
handler can also return 400 Bad Request and 409 Conflict

Reply:
The comment was just meant to point out the previous two success conditions, but it's kind of obvious now that we removed accepted status code.